oracle/price-feeder/oracle/oracle.go

```diff
@@ -382,9 +411,14 @@ func GetComputedPrices(
 			return nil, err
 		}
 
-		return vwapPrices, nil
+		for asset, price := range vwapPrices {
+			if _, ok := tvwapPrices[asset]; !ok {
+				tickerAssets = append(tickerAssets, asset)
+				tvwapPrices[asset] = price
```

Comment:
small nit:  i wonder if there's a better name for `tvwapPrices` if it can contain a vwapPrice

Reply:
we can call it computedPrices